test/definitions/apm/.net/windows/windows2019-iis.json

```diff
@@ -0,0 +1,78 @@
+{
+  "global_tags": {
+      "owning_team": "OpenSource",
+      "Environment": "development",
+      "Department": "Product",
+      "Product": "Virtuoso"
+  },
+
+  "resources": [{
+      "id": "iiswindows2019",
+      "display_name": "IisWindows2019Host",
+      "provider": "aws",
+      "type": "ec2",
+      "size": "t3a.small",
+      "is_windows": true,
+      "ami_name": "Windows_Server-2019-English-Full-HyperV-*",
+      "user_name": "Administrator"
+  }],
+
+  "services": [{
+    "id": "iis",
+    "destinations": ["iiswindows2019"],
+    "source_repository": "https://github.com/newrelic/open-install-library.git",
+    "deploy_script_path": "test/deploy/windows/iis/install/roles",
+    "port": 9999
+  },{
+    "id": "webapp1",
+    "display_name": "Web Application 1",
+    "destinations": ["iiswindows2019"],
+    "source_repository": "https://github.com/newrelic/open-install-library.git",
+    "deploy_script_path": "test/deploy/windows/iis/deploy-webapplication/roles",
+    "port": 8081
+  },{
+    "id": "webapp2",
+    "display_name": "Web Application 2",
+    "destinations": ["iiswindows2019"],
+    "source_repository": "https://github.com/newrelic/open-install-library.git",
+    "deploy_script_path": "test/deploy/windows/iis/deploy-webapplication/roles",
+    "port": 8082
+  },{
+    "id": "webapp3",
+    "display_name": "Web Application 3",
+    "destinations": ["iiswindows2019"],
+    "source_repository": "https://github.com/newrelic/open-install-library.git",
+    "deploy_script_path": "test/deploy/windows/iis/deploy-webapplication/roles",
+    "port": 8083,
+    "params": {
+      "is_core_app": "true"
+    }
+  }],
+
+  "instrumentations": {
+    "resources": [
+      {
+          "id": "nr_infra",
+          "resource_ids": ["iiswindows2019"],
+          "provider": "newrelic",
+          "source_repository": "https://github.com/newrelic/open-install-library.git",
+          "deploy_script_path": "test/deploy/windows/newrelic-cli/install-recipe/roles",
+          "params": {
+              "recipe_content_url": [
+                  "https://raw.githubusercontent.com/newrelic/open-install-library/main/recipes/newrelic/apm/.net/windows/net-install.yml"
+              ]
+          }
```

Comment:
Just food for thought: maybe have a separate test that installs both infra agent and .NET agent? I'm mostly curious to know if that would "just work".

Reply:
Yes, once we clarified our valid installs path (only stitch path?).